spec/latest/core/index.html

```diff
@@ -557,8 +557,9 @@
     <p>Keep in mind that operations specified in an <i>ApiDocumentation</i>
       may fail at runtime as either resources or the <i>ApiDocumentation</i>
       itself have changed since they have been retrieved. A simple strategy
-      to try to recover from such an error is to reload the
-      <i>ApiDocumentation</i>.</p>
+      to avoid such an error without affecting performance is to deliver the 
+      <i>ApiDocumentation</i> with "Last-Modified" cache-info and require the 
```

Comment:
I don't think we should recommend "Last-Modified" over "Etag", I also don't think that even with suggestion to rely on HTTP caching we don't need any more original paragraph with recommendation for how to recover from error.